apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -864,140 +1298,86 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
-                                disabled={ isSubmitting }
-                            >
-                                { t("common:close") }
-                            </LinkButton>
-                        </Grid.Column>
-                        { !showResponseView || isSubmitting ? (
-                            <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
-                                <PrimaryButton
-                                    data-testid={ `${componentId}-finish-button` }
-                                    data-componentid={ `${componentId}-finish-button` }
-                                    floated="right"
-                                    onClick={ handleBulkUserImport }
-                                    loading={ isSubmitting }
-                                    disabled={ isLoading || isSubmitting ||  hasError || !selectedCSVFile }
-                                >
-                                    { t("console:manage.features.user.modals.bulkImportUserWizard.buttons.import") }
-                                </PrimaryButton>
-                            </Grid.Column>
-                        ) : null }
-                    </Grid.Row>
+                    {
+                        configureMode == MultipleInvites.MANUAL
+                            ? (
+                                <Grid.Row column={ 1 }>
+                                    <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
+                                        <LinkButton
+                                            data-testid={ `${componentId}-cancel-button` }
+                                            floated="left"
+                                            onClick={ () => {
+                                                closeWizard();
+                                                setshowManualInviteTable(false);
+                                            } }
+                                            disabled={ isSubmitting }
+                                        >
+                                            { t("common:close") }
+                                        </LinkButton>
+                                    </Grid.Column>
+                                    { !showManualInviteTable || isSubmitting ? (
+                                        <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
+                                            <PrimaryButton
+                                                data-testid={ `${componentId}-finish-button` }
+                                                floated="right"
+                                                onClick={ manualInviteMultipleUsers }
+                                                loading={ isSubmitting }
+                                                disabled={ isLoading || isSubmitting || hasError }
+                                            >
+                                                Invite
```

Comment:
Shall we use the i18n string here?

Reply:
Fixed with 899a8e06a8a09eccfe9a0f5feb3405a95bcae33e.